spec/rubocop/cop/style/inverse_methods_spec.rb

```diff
@@ -160,25 +145,26 @@ def test_method
   it_behaves_like 'all variable types', 'foo["bar"]'
   it_behaves_like 'all variable types', 'foo.bar'
 
-  { any?: :none?,
+  {
```

Comment:
Since this test doesn't use safe navigation, why is `any?: :none?` removed?

Reply:
Oops. I mistakenly remove it. Thanks for pointing it out.